cmd/destroy/destroy.go

```diff
@@ -189,23 +182,21 @@ func Destroy(ctx context.Context) *cobra.Command {
 			os.Setenv("KUBECONFIG", kubeconfigPath)
 			defer os.Remove(kubeconfigPath)
 
-			// when option --all the cmd will destroy everything at the namespace and return
-			if options.DestroyAll {
-				if !okteto.Context().IsOkteto {
-					return oktetoErrors.ErrContextIsNotOktetoCluster
-				}
-				err = c.runDestroyAll(ctx, options)
-				if err == nil {
-					oktetoLog.Success("All resources at namespace '%s' where successfully destroyed", options.Namespace)
-				}
-			} else {
-				err = c.runDestroy(ctx, options)
-				if err == nil {
-					oktetoLog.Success("Development environment '%s' successfully destroyed", options.Name)
+			manifest, err := model.GetManifestV2(options.ManifestPath)
```

Comment:
This affects how the destroy all works. When the flag `--all` is set, it doesn't affect to any specific manifest, so doing this before the `destroy-all` might generate errors.


In the description you mention something about fixing the destroy all, you mean something related to this?

Maybe we should check the flag `--all` first? That is always executed remotely

Reply:
To know if destroy command needs to be executed remotely or not we need to know previously some values in the manifest (check if destroy image is set) . Condition -> https://github.com/okteto/okteto/blob/adrian/deploy-destroy-same-way/cmd/destroy/destroy.go#L229. The fix I am referring to is to update with master because there is a fix that was uploaded for destroy all. I didn't want to include it in this PR, I think fits better to update later `feature/remote-deploy` branch with master's fix once this PR is merged.

Reply:
If there is a `--all` flag, it follows other flow. With that flag, the destroy already works remotely as it runs on Okteto backend.

Including the logic to `GetManifest` can be confusing when following the code or even checking the logs and it could be "error prone" in the future if we start to handle the error at some point in the future

Reply:
yeah, you are right. It can be confusing debugging `destroy --all` if we read the manifest. I will add an exception for this flag 👍